components/card/profile/src/index.scss

```diff
@@ -0,0 +1,76 @@
+@import '../node_modules/@schibstedspain/sui-theme/lib/index';
+@import '../node_modules/@schibstedspain/sui-rating-icons/lib/index';
+@import 'settings';
+
+
+.sui-CardProfile {
+  background-color: $bgc-card-profile;
+  display: flex;
+  flex-direction: column;
+  height: $w-card-profile;
+  width: $h-card-profile;
+
+  &-topcontainer {
+    display: flex;
+    flex-grow: .4;
+
+    &Avatar {
+      &Halo {
+        border: $bd-card-profile-avatar-halo;
+        border-radius: 50%;
+        height: ($h-card-profile-avatar + $p-card-profile-avatar);
+        margin: $m-l;
+        width: ($w-card-profile-avatar + $p-card-profile-avatar);
+      }
+
+      &RadiusWrapper {
+        border-radius: 50%;
+        display: block;
+        height: $w-card-profile-avatar;
+        margin-left: auto;
+        margin-right: auto;
+        margin-top: $m-s;
+        overflow: hidden;
+        width: $w-card-profile-avatar;
+
+        img {
```

Comment:
i would suggest to add a class instead of point the html tag.

for instance: `sui-CardProfile-avatarImage`